### PR TITLE
chore: add tags on resources

### PR DESCRIPTION
**Pull request by luislhl**
### Acceptance Criteria
- Add tags on the AWS resources created with serverless. The tags are `Application` and `Stage`.

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -228,14 +233,14 @@ functions:
...
-          arn:aws:lambda:eu-central-1:769498303037:function:hathor-explorer-service-${self:custom.explorerServiceStage}-create_or_update_dag_metadata
+          arn:aws:lambda:eu-central-1:${self:provider.environment.ACCOUNT_ID}:function:hathor-explorer-service-${self:custom.explorerServiceStage}-create_or_update_dag_metadata
   loadWalletAsync:
     handler: src/api/wallet.loadWallet
     warmup:
       walletWarmer:
         enabled: false
   loadWalletApi:
-    role: arn:aws:iam::769498303037:role/WalletServiceLoadWalletLambda
+    # role: arn:aws:iam::${self:provider.environment.ACCOUNT_ID}:role/WalletServiceLoadWalletLambda
```
Comment (andreabadesso) [inline serverless.yml]: Should this be commented?

**Review by andreabadesso — CHANGES_REQUESTED** [review]

**Review by andreabadesso — APPROVED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -37853,6 +37899,11 @@
       "integrity": "sha512-cwESVXlO3url9YWlFW/TA9cshCEhtu7IKJ/p5soJ/gGpj7vbvFrAY/eIioQ6Dw23KjZhYgiIo8HOs1nQ2vr/oQ==",
       "dev": true
     },
+    "user": {
+      "version": "0.0.0",
```
Comment (alexruzenhack) [inline package-lock.json]: @luislhl This is strange. I nerve saw a package versioned like this.

**Review by alexruzenhack — APPROVED** [review]
I have only one comment about the package `"user": "0.0.0"`. This is a strange value. Does it worth double check on it?

File: package-lock.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "hathor-wallet-service",
-      "version": "1.22.0-alpha",
+      "version": "1.22.3-alpha",
```
Comment (alexruzenhack) [inline package-lock.json:9]: Thank you!

**Review by alexruzenhack — APPROVED** [review]
Nice!

File: package-lock.json (hunk 2 above)
Comment (luislhl) [reply package-lock.json]: I've removed this plugin. I was led to believe we needed it by their own page on the serverless site: https://www.serverless.com/plugins/serverless-plugin-resource-tagging

But, after your comment, I decided to make sure and it turns out serverless already has a good built-in support for tagging resources.

**Review by andreabadesso — APPROVED** [review]

**Pull request merged**